src/api/todoApi.js

```diff
@@ -23,7 +23,7 @@ const todos = [
 ];
 
 class TodoApi {
-    add(todo){
+    add(todo) {
         todos.push(todo);
```

Comment:
You will probably need to generate an id here in order to better track where to add / delete todos
```js
{ id: "some-max-id + 1", ...todo}
```